packages/css/src/components/description-list/description-list.scss

```diff
@@ -33,20 +32,22 @@
 }
 
 .ams-description-list__term {
+  font-weight: var(--ams-description-list-term-font-weight);
+
   @media screen and (min-width: $ams-breakpoint-medium) {
     grid-column-start: 1;
   }
 }
 
-@mixin reset-details {
+@mixin reset-dd {
```

Comment:
We're using `dd`, `details` (in the class name) and `description` (in the README) now. Shouldn't it all be `description`?

Reply:
Agreed – you may have missed this in the PR description:

> We should rename `DescriptionList.Details` to `DescriptionList.Description` and the ‘details’ tokens and classes. Let’s do this in a separate PR.

I propose using the element name in our mixins.

Reply:
Ah, missed that part. A separate PR for `Details` -> `Description` sounds like a good idea to me.

> I propose using the element name in our mixins.

That would be a new convention, right? Makes sense to me, but maybe we should do that in a separate PR for all relevant mixins, instead of just here?

Reply:
Sure, although we could have let this one slip in already? Reverted it.